ibis/backends/tests/test_string.py

```diff
@@ -3,6 +3,7 @@
 import contextlib
 
 import pandas as pd
+import pyspark
```

Comment:
`pyspark` is not a required dependency so you need to use the pattern we use elsewhere in the codebase to avoid importing it if it isn't installed:

```python
try:
    from pyspark.sql.utils import PythonException
except ImportError:
    PythonException = None
```